spec/models/miq_expression/miq_expression_spec.rb

```diff
@@ -1,52 +1,52 @@
 describe MiqExpression do
   describe "#to_sql" do
     it "generates the SQL for an EQUAL expression" do
-      sql, * = MiqExpression.new("EQUAL" => {"field" => "Vm-name", "value" => "foo"}).to_sql
-      expect(sql).to eq("vms.name = 'foo'")
+      sql, * = MiqExpression.new("EQUAL" => {"field" => "Vm-name", "value" => "Foo"}).to_sql
+      expect(sql).to eq("LOWER(vms.name) = 'foo'")
     end
 
     it "generates the SQL for a = expression" do
-      sql, * = MiqExpression.new("=" => {"field" => "Vm-name", "value" => "foo"}).to_sql
-      expect(sql).to eq("vms.name = 'foo'")
+      sql, * = MiqExpression.new("=" => {"field" => "Vm-name", "value" => "Foo"}).to_sql
+      expect(sql).to eq("LOWER(vms.name) = 'foo'")
     end
 
     it "generates the SQL for a LIKE expression" do
-      sql, * = MiqExpression.new("LIKE" => {"field" => "Vm-name", "value" => "foo"}).to_sql
-      expect(sql).to eq("vms.name LIKE '%foo%'")
+      sql, * = MiqExpression.new("LIKE" => {"field" => "Vm-name", "value" => "Foo"}).to_sql
+      expect(sql).to eq("LOWER(vms.name) LIKE '%foo%'")
```

Comment:
ILIKE seems more appropriate (instead of downcase of parameter and field) when generating sql with ‘LIKE’. However, it would require more refactoring.  Since massive refactoring is already in our queue, may be it make sense to implement ILIKE on later phases?